src/doc/building_visit/BuildingOnWindows/ConfiguringWithCMakeGUI.rst

```diff
@@ -48,9 +48,7 @@ generation should locate this directory automatically.
 
     Directory structure with source from a released version
 
-For developement build cloned or downloaded from the github repositories, in
-order for CMake to locate the directory automatically, visit-deps should be 
-peer to visit. 
+In order for CMake to locate the directory automatically for development build cloned or downloaded from the github repositories, *visit-deps* should be peer to *visit*. 
```

Comment:
Part of the sentence near "for development build cloned or downloaded" doesn't read quite right. Maybe change to "for a development build cloned or downloaded".

Reply:
done.